### PR TITLE
chore: moved GTM into signup

### LINKED ISSUE #14153: dont use google analytics for embeds

### PR DESCRIPTION
**Pull request by PeerRich**
- [ ] clean up cookie after signup
- [ ] test if tracking only shows up on signup and not in other parts of the app

### CODE REVIEW

**Review by sean-brydon — CHANGES_REQUESTED** [review]
I'll take this over next week - i think we need to follow full GMv2 compliance here

File: apps/web/pages/signup.tsx [hunk 1]
```diff
@@ -247,319 +254,349 @@ export default function Signup({
...
-            ) : (
-              <p className="text-subtle text-base font-medium leading-5">
-                {t("calcom_explained", {
-                  appName: APP_NAME,
-                })}
-              </p>
+    <>
+      {IS_CALCOM && COOKIE_CONSENT && process.env.NEXT_PUBLIC_GTM_ID ? (
+        <>
+          wasd
```
Comment (sean-brydon) [inline apps/web/pages/signup.tsx:260]:
```suggestion
```

**Review by sean-brydon — DISMISSED** [review]
Let’s address the changes we mentioned in threads on Monday -

**Review by zomars — APPROVED** [review]
Fixed conflicts and approved

**Pull request merged**